### ISSUE TITLE
Restoration choice of "Create new backup" or "Reuse" can be bypassed

### ISSUE THREAD

**Issue by t-m-w** [stevesoltys/backup#789]
The [intended behavior](https://github.com/seedvault-app/seedvault/pull/785#issuecomment-2450680172) is to ask the user the question about whether they want to create a new backup or reuse the existing backup, until they choose. Currently, it can be bypassed.

1. Restore a backup.
2. When asked the question, either press the Back button if you're in the Setup Wizard, or close Seedvault (swipe it out of Overview/Recents).
3. Open Seedvault again.
4. Turn on backups.
5. Choose Backup Now.

Expected result: At some point after step 2, you must make the choice.

Actual result: You never need to make the choice. The backup proceeds anyway.

**Comment by grote** [comment]
Wait, so you started by restoring a backup? from SUW? Then, the question shouldn't be asked at all, especially not in SetupWizard.

**Comment by t-m-w** [comment]
> Wait, so you started by restoring a backup? from SUW? Then, the question shouldn't be asked at all, especially not in SetupWizard.

Yes, but then when should it be asked? I've thought before that it may make more sense to ask when the user turns on backups.

**Comment by grote** [comment]
Ah I confused the special re-use screen and the dialog that comes up when starting a new installation of seedvault for the first time. You are talking about that entire re-use backup screen, right?

Where do you land when pressing the back button there?

I guess the observed behavior is alright. Closing or backing our is equivalent to not re-using. Safer course of action.

**Comment by t-m-w** [comment]
> You are talking about that entire re-use backup screen, right? Where do you land when pressing the back button there?

Yes. I end up back at the Setup Wizard's page that asks if I want to restore a backup at all, and I can just Skip that.

**Comment by grote** [comment]
ok, then I guess it is ok that this page can be skipped and what you saw is the expected behavior after all? case closed?
